### DISCUSSION TITLE
[Feature] Sort by size option

### DISCUSSION THREAD

**Discussion by othyn** [immich-app/immich#11215] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'd love an option to sort a given view, or the main timeline, by file size. Its a really useful tool to help cleanup large unwanted files.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#10400

**Reply by Metacowboy** [reply]
Large Option Search is nice , but a Size Search would help to sort out all the Thumbnails some Software generate when you capture video and easyli get mixed as dubblets.

**Comment by Kieros** [comment]
not only that I have also lots of thumbnails that were uploaded by accident. From the main view they don't look that bad untill you open them.

Would be nice to sort on size so all thes 50kb files can be found more easely.
